protokube/pkg/protokube/vsphere_volume.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package protokube
+
+import (
+	"github.com/golang/glog"
+	"net"
+)
+
+const EtcdDataKey = "01"
+const EtcdDataVolPath = "/mnt/master-" + EtcdDataKey
+const EtcdEventKey = "02"
+const EtcdEventVolPath = "/mnt/master-" + EtcdEventKey
+
+// TODO Use lsblk or counterpart command to find the actual device details.
+const LocalDeviceForDataVol = "/dev/sdb1"
+const LocalDeviceForEventsVol = "/dev/sdc1"
+const AttachedToValue = "localhost"
+const VolStatusValue = "attached"
+const EtcdNodeName = "a"
+const EtcdClusterName = "main"
+const EtcdEventsClusterName = "events"
+
+type VSphereVolumes struct {
+	paths map[string]string
+}
+
+var _ Volumes = &VSphereVolumes{}
+
+func NewVSphereVolumes() (*VSphereVolumes, error) {
+	vsphereVolumes := &VSphereVolumes{
+		paths: make(map[string]string),
+	}
+	vsphereVolumes.paths[EtcdDataKey] = EtcdDataVolPath
+	//vsphereVolumes.paths[EtcdEventKey] = EtcdEventVolPath
```

Comment:
Do you plan to checkin like this? If yes, can you put a comment on why this line is commented or remove the commented line